### ISSUE TITLE
Deploying to a Dev Sandbox is not possible due to "default" namespace being picked up during deployment

### ISSUE THREAD

**Issue by odockal** [containers/podman-desktop#2562]
### Bug description

It is not directly possible to app deploy to RH Dev sandbox. There is a `default` namespace chosen during deploy to kubernetes workflow. One has to manually update kubeconfig file and add `namespace: user-dev` in order to make it run. 

### Operating system

Windows 10

### Version

1.0.0

### Steps to reproduce

1. Install dev sandbox ext.
2. Start podman
3. Login to a Dev sandbox and get login command
4. Create Dev Sandbox connection
5. Pull an image (`quay.io/centos7/httpd-24-centos7`)
6. Create a container
7. Deploy to Kubernetes - choose right context (sandbox), check to create route
8. Assert: pod is deployed to cluster, route is created, pod is running, app is available
Actual: error appears. pod is not created. One has to manually update kubeconfig file to utilize user available namespace in Dev Sandbox

### Relevant log output

```shell
Pod creation status: Error: pods is forbidden: User "username" cannot create resource "pods" in API group "" in the namespace "default"
```

**Comment by dgolovin** [comment]
This is fixed in sandbox extension available in latest build.

**Comment by odockal** [comment]
When approximately that released happened? I was just trying (around 9 PM CET) and it worked out of box. Was installation of devsandbox ext. pulling that release?

**Comment by slemeur** [comment]
https://github.com/redhat-developer/podman-desktop-sandbox-ext/issues/7

**Comment by odockal** [comment]
I am still seeing this issue. Cannot we release a fix?

**Comment by dgolovin** [comment]
the fix did not get into v0.0.1 release, v0.0.2 should be released using `main` branch.

**Comment by dgolovin** [comment]
Follow https://github.com/redhat-developer/podman-desktop-sandbox-ext/issues/7. 
Closing.
